py/torch_tensorrt/dynamo/conversion/impl/unary/base.py

```diff
@@ -9,6 +12,23 @@
 )
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor, TRTUnaryOp
+from typing import Union, Callable, Any, Optional
+
+SUPPORTED_ATEN_TORCH_UNARY_OPS: Dict[
+    TRTUnaryOp, Union[Callable[[Any], Any], Callable[[Any, Any], Any]]
+] = {}
+
+
+def get_python_op_from_trt_unary_op(
+    trt_op: TRTUnaryOp,
+) -> Union[Callable[[Any], Any], Callable[[Any, Any], Any]]:
+    if trt_op == trt.UnaryOperation.SQRT:
+        SUPPORTED_ATEN_TORCH_UNARY_OPS[trt_op] = partial(operator.pow(a, b=0.5))
+        return True
+    else:
+        raise RuntimeError(f"{trt_op} is not supported yet!")
+        return False
```

Comment:
Could change the error message to `f"Applying {trt_op} to non-ITensor inputs is not yet supported."`, to make it clear that the lack of support is for applying the op to a non-ITensor input and not generally.

Additionally, the return could be omitted here.

Reply:
Yes will make this change.